src/main/java/core/basesyntax/file/operations/FIleWriterImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.file.operations;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class FIleWriterImpl implements FileWriter {
```

Comment:
The class name `FIleWriterImpl` is incorrectly capitalized. It should be `FileWriterImpl` to match Java naming conventions and ensure consistency across the project.